examples/riddle-prep-buddy-v2/README.md

```diff
@@ -38,15 +38,25 @@ A small tkinter application students can use to practice for the riddles section
 
 ## Running the Application
 
-1. **Run the Server Notebook:**
+1. **Access to Server notebook:**
 
-    Open the `[Release] Riddle Prep App Server.ipynb` file, preferably in Colab if you don't have a GPU on your, follow the instructions to start the server, and copy the `ngrok` URL.
+    Open the `Riddle_Prep_App_Server.ipynb` file, preferably in Colab if you don't have a GPU on your, follow the instructions to start the server, and copy the `ngrok` URL.
 
-2. **Update `app.py`:**
+    You may also find the same notebook via this link: `https://colab.research.google.com/drive/1staK1vtcI9ZMont9rxazCPn8E1nm1R5c?usp=sharing`
+
+2. **Running the Server:**
+    a. You will first of all need access to the TTS model. This is only accessible to Hackathon participants.
```

Comment:
please qualify is to say `AfricAEID Hackathon` for future readers of this doc

Reply:
@wsedor, I have fixed the issues you pointed out.
- Renamed the server notbook (and all reference to it) to `Riddle_Prep_App_Server.ipynb`
- Fixed the incomplete sentence.
- Added AfricaAIED Hackathon qualifier to all instances referencing the Hackathon.

Reply:
@wsedor, kindly review when you see this.
P.S.: The changes can be found in the latest commit from yesterday, "fix issues in README.md".